react/README.md

```diff
@@ -9,6 +9,277 @@ For migration between style guides, please see [MIGRATION.md](MIGRATION.md).
 1.  [Creation of a new project](#creation-of-a-new-project)
 2.  [Different kinds of components](#different-kinds-of-components)
 3.  [File naming](#file-naming)
+4.  [Translations](#translations)
+4.  [Folder structure](#folder-structure)
+5.  [Tools](#tools)
+
+---
+
+## Creation of a new project
+
+[Create-react-app](https://github.com/facebook/create-react-app) should always be the starting project of any new ReactJS project.
+
+If you're unsure about the tools used or ReactJS generally, then you shouldn't be the one setting up the project or initializing the different tools. Ask someone who has experience. It's often easier to see how someone else does something and try to mimic that instead of starting on your own and trying to follow this style guide.
+
+---
+
+## Different kinds of components
+
+
+### Use-case
+
+Your web application will contain several use-cases, which in their turn can contain different screens. A use-case can, for example, be a 'Users'. This use-case manages the creation, editing, deletion, and viewing of users. This is a neat way of structuring your project.
+
+### Screen
+
+A screen is a component that takes up the whole screen, except the navigation. It may contain [components](#components), but it may not contain a [use-case](#use-case).
+
+Two examples for what screens can be:
+
+-   Login: A simple screen that contains a component LoginForm.
+-   ShowUsers, UserDetail: Two screens that are a part of the [use-case](#use-case) Users.
+
+
+### Component
+
+A component is everything that's not a [use-case](#use-case) or [screen](#screen). The only important thing to remember is to give a component only responsibility for one thing. In other words, follow the Single-responsibility principle.
+
+---
+
+## File naming
```

Comment:
You don't mention that files can be named `Element.comp.jsx` but include them in the example below.

Reply:
I changed it to this and committed:

```md
## File naming

Each folder, including use-cases, screens, and components, can, but must not, contain these types of files:

* `index.js`, should contain the export of the `*.jsx` file. If there's a container, however, it should export that instead. index.js is used to you only have to import the folder, not a specific file inside the folder.

* `MyUseCase.jsx` / `MyScreen.screen.jsx` / `MyComponent.comp.jsx`: the main element.

* `Element.translations.json`/`Element.(screen/comp).translations.json`: has the translation data for the Element.
```

Reply:
![image](https://user-images.githubusercontent.com/7045380/112178610-30b22c00-8bfa-11eb-9af2-d9b7ecac4cec.png)